combat-trainer.lic

```diff
@@ -213,6 +213,9 @@ class LootProcess
     @ritual_type = thanatology['ritual_type'].downcase
     echo("  @ritual_type: #{@ritual_type}") if $debug_mode_ct
 
+    @cycle_rituals = false
+    @cycle_rituals = @ritual_type == "cycle"
```

Comment:
Oh. Derp. That makes sense. @ritual_type == "cycle" returns true or false which is then assigned to @cycle_rituals via =.  Does that sound accurate?